restricting-traffic/README.md

```diff
@@ -15,6 +15,8 @@ For background information on the security controls used, see [Anthos security b
 
 ## Denying traffic unless it is explicitly allowed
 
+These instructions are for GKE clusters using Dataplane V2 or clusters not using NodeLocal DNScache. If you have GKE clusters using NodeLocal DNScache but not using Dataplane V2 (including GKE on prem) follow the instructions in README-nodelocaldns.md instead. For mixed environments follow both sets of instructions.
```

Comment:
Maybe something on how to determine which you have?

Reply:
added commands, PTAL